src/app/current-users/current-users.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <label><h1>USERS</h1></label>
+  <table *ngIf="usersTable.length !== 0 else notFound" class="usersTable" border="1">
+    <thead>
+      <th>Name</th>
+      <th>Username</th>
+      <th>Email</th>
+      <!-- <th>Authority Type</th> -->
+      <th>Delete User</th>
+      <th>Update Username</th>
+    </thead>
+    <tbody>
+      <tr *ngFor="let user of usersTable | slice:(pageCounter * 10): (pageCounter + 1) * 10">
+        <td>{{user.name}}</td>
+        <td>{{user.username}}</td>
+        <td>{{user.email}}</td>
+        <!-- <td>{{user.authorityType}}</td> -->
+        <td><button (click)="deleteUser(user)" class="deletebtn">Delete</button></td>
+        <td>
+          <form class="form-inline" #f="ngForm" (ngSubmit)="updateUser(f,user)">
+            <input type="text" name="username" ngModel #username="ngModel" placeholder="Enter new Username..." required>
```

Comment:
Aesthetically, I'm not entirely a fan of the input text field here. But I think this will definitely work for our demo.

Reply:
Agreed, I will definitely change it to something better looking later on.